src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -90,130 +78,60 @@ WMS_Provider.prototype.tileInsideLimit = function tileInsideLimit(tile, layer) {
     return tile.level > 2 && layer.bbox.intersect(tile.bbox);
 };
 
-WMS_Provider.prototype.getColorTexture = function getColorTexture(tile, layer, bbox, pitch) {
+WMS_Provider.prototype.getColorTexture = function getColorTexture(tile, layer) {
     if (!this.tileInsideLimit(tile, layer) || tile.material === null) {
         return Promise.resolve();
     }
 
-    var url = this.url(bbox.as('EPSG:4326'), layer);
-
-    var result = { pitch };
-    result.texture = this.cache.getRessource(url);
-
-    if (result.texture !== undefined) {
-        return Promise.resolve(result);
-    }
+    const url = this.url(tile.bbox.as('EPSG:4326'), layer);
+    const pitch = new THREE.Vector3(0, 0, 1);
+    const result = { pitch };
 
-    const { texture, promise } = Fetcher.texture(url);
-    result.texture = texture;
-
-    result.texture.generateMipmaps = false;
-    result.texture.magFilter = THREE.LinearFilter;
-    result.texture.minFilter = THREE.LinearFilter;
-    result.texture.anisotropy = 16;
-
-    return promise.then(() => {
-        this.cache.addRessource(url, result.texture);
-        result.texture.needsUpdate = true;
+    return this.getColorTextureByUrl(url).then((texture) => {
+        result.texture = texture;
+        result.texture.coordWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
```

Comment:
WMS is tiled and works like WMTS.
I need to know his correspondence with tile coordinate